tools/downloader/requirements-pytorch.in

```diff
@@ -1,4 +1,4 @@
 onnx
 scipy           # via torchvision
-torch
+torch==1.1      # required to convert upsample_bilinear2d
```

Comment:
The model can be converted with all PyTorch, except 1.2. Could you test it with >1.2?

Reply:
requires torch>=1.4
I update the req. file accordingly.